frontend/pulse3d/src/pages/upload-form.js

```diff
@@ -176,7 +178,7 @@ export default function UploadForm() {
         usageQuota.limits &&
         parseInt(usageQuota.limits.jobs) !== -1
     );
-  }, [paramErrors, files, inProgress]);
+  }, [paramErrors, files, inProgress, wellGroupErr]);
```

Comment:
Why isn't `wellGroupErr` just another possible error under `paramErrors`?

Reply:
Because the structure is different and became overly complicated with handling errors for both the label name input and the well input. it would have required changes to how the current `paramErrors` get checked as opposed to checking this extra booleann value

Reply:
If you think it would be better in paramErrors, I can refactor to add it in

Reply:
Ideally we should have just one place to check for errors in the analysis form, but if it's going to take a significant refactor we can do it later